### ISSUE TITLE
auto eat issue

### ISSUE THREAD

**Issue by Danmark3110** [Zergatul/cheatutils#111]
playing TNP Limitless 7 and auto eat is not eating infinity ham ;( or im just dumb

**Comment by Zergatul** [comment]
Does it work with vanilla food items?

**Comment by Danmark3110** [comment]
it works with normal and modded but not that

**Comment by Zergatul** [comment]
cheatutils relies on food data property of the item you are holding in offhand. If this property is missing, mod assumes you are not holding food and don't try to use this item. What is this infinity ham? Maybe something very custom and mod specific?

**Comment by Danmark3110** [comment]
its from the mod relics its just food that doesnt run out basiclly it doesnt get consumed maybe thats why?

**Comment by Zergatul** [comment]
Yes... I checked their source code, and this is how they define infinity ham item properties:
https://github.com/Octo-Studios/relics/blob/1.21.0/src/main/java/it/hurts/sskirillss/relics/items/relics/InfinityHamItem.java#L46
```java
new Item.Properties()
    .stacksTo(1)
    .rarity(Rarity.RARE)
```
And this is how porkchop properties is defined in vanilla Minecraft:
```java
new Item.Properties()
    .food(Foods.PORKCHOP)
```
There is no "food" property on infinity ham.

**Comment by Zergatul** [comment]
You can probably code your own auto eat from events scripting

**Comment by Danmark3110** [comment]
i do not have the iq for that

**Comment by Zergatul** [comment]
are you in 1.21+ version?

**Comment by Danmark3110** [comment]
1.20.1 cant lie this is a slow way of texting

**Comment by Zergatul** [comment]
1.20.1 probably lacks some features to code this

**Comment by Danmark3110** [comment]
bummer
